packages/server/src/utils/validate-env.ts

```diff
@@ -12,6 +12,7 @@ export function validateEnv(): void {
     SESSION_DURATION: num(),
     EMAIL_VERIFICATION_EXPIRATION_TIME: num(),
     PASSWORD_RESET_EXPIRATION_TIME: num(),
+    MOBILE_VERIFICATION_EXPIRATION_TIME: num(),
```

Comment:
Do we actually need a separate value for this one? Maybe instead we can use some generic `VERIFICATION_EXPIRATION_TIME`?

Reply:
I think email notification and mobile notification should have separate expiration time. For example 12 hourse and 5 minutes accordingly.